index.js

```diff
@@ -4,32 +4,55 @@ var postcss = require('postcss'),
 	glob = require('glob'),
 	path = require('path');
 
-function processArgs(nodes) {
+function processArgs(nodes, functions) {
 	var args = [];
 	var last = nodes.reduce(function (prev, node) {
 		if (node.type === 'div' && node.value === ',') {
 			args.push(prev);
 			return '';
 		}
-		return prev + valueParser.stringify(node);
+		if (node.type !== 'function' || !functions.hasOwnProperty(node.value)) {
+			return prev + valueParser.stringify(node);
+		}
+		return transformFunction(node, functions)
+			.then(function (result) {
+				return prev + result;
+			});
 	}, '');
 	if (last) {
 		args.push(last);
 	}
 
-	return args;
+	return Promise.all(args);
+}
```

Comment:
@TrySound what do you mean? It has to wait until all the calls of `transformFunction` are resolved before returning the result.